editoast/src/main.rs

```diff
@@ -771,13 +774,21 @@ mod tests {
         // GIVEN
         let rolling_stock_name =
             "fast_rolling_stock_import_non_electric_rs_without_startup_and_panto_values";
-        let mut non_electric_rolling_stock = get_fast_rolling_stock(rolling_stock_name);
-        if let Some(ref mut effort_curves) = non_electric_rolling_stock.effort_curves {
-            effort_curves.modes.remove("25000V");
-            non_electric_rolling_stock.electrical_power_startup_time = Some(None);
-            non_electric_rolling_stock.raise_pantograph_time = Some(None);
-        }
-        let file = generate_temp_file(&non_electric_rolling_stock);
+        let non_electric_rs_changeset = get_fast_rolling_stock(rolling_stock_name);
+
+        let effort_curves = non_electric_rs_changeset
+            .effort_curves
+            .clone()
```

Comment:
```suggestion
            .as_mut()
```

maybe that works and avoids the cloning